src/dbnode/generated/proto/namespace/namespace.proto

```diff
@@ -33,6 +33,31 @@ message NamespaceOptions {
     bool coldWritesEnabled                          = 10;
     NamespaceRuntimeOptions runtimeOptions          = 11;
     google.protobuf.BoolValue cacheBlocksOnRetrieve = 12;
+    AggregationOptions aggregationOptions           = 13;
+}
+
+message AggregationOptions {
+    repeated Aggregation aggregations = 1;
```

Comment:
why multiple aggregation options? I thought each namespace was a different aggregation level

Reply:
Yeah, generally that's true. The `repeated` nature of this field helps us account for the occasional case where we want to send aggregated metrics to a namespace that's also receiving unaggregated metrics. This can be accomplished via rollup rules (see the note at the bottom here: https://docs.m3db.io/operational_guide/mapping_rollup/).

Reply:
May be worth adding comments on this field describing these common configurations that justify allowing multiple aggregations per namespace.

Reply:
Yeah, not a bad idea. I commented the types pretty heavily in the `namespace/types.go` file but will pull some of that over to the proto file as well.